conf/application.conf

```diff
@@ -21,6 +21,8 @@ appName = income-tax-interest
 
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
+play.modules.enabled += "config.Module"
```

Comment:
You have not added the corresponding Module that this is trying to enable. See - https://github.com/hmrc/income-tax-state-benefits/blob/902b9a780221a9821acd826d214f027a51394ff8/app/config/Module.scala#L27-L26